spec/requests/v1/notice_of_disagreements_controller_request_spec.rb

```diff
@@ -16,18 +16,40 @@
 
   before { sign_in_as(user) }
 
+  # add tests for rollback
   describe '#create' do
     def personal_information_logs
       PersonalInformationLog.where 'error_class like ?',
                                    'V1::NoticeOfDisagreementsController#create exception % (NOD_V1)'
     end
 
+    # Workaround to get the tests running
+
+    # TODO: clean this up after vets-json-schema is updated
+    # let(:test_request_body) do
+    #   ** This (below) looks more like the response body than the request body based on LH docs **
+    #   example_data = VetsJsonSchema::EXAMPLES.fetch('NOD-CREATE-REQUEST-BODY_V1')
+    #   example_data['data']['attributes']['formData']['nodUploads'] = [
+    #     {
+    #       "name": "my_file.pdf",
+    #       "confirmationCode": "aaad6f52-2e85-43d4-a5a3-1d9cb4e482a0"
+    #     }
+    #   ]
+    #   example_data
+    # end
+
+    # According to docs, this is what the structure of the request body for the v2 endpoint
     subject do
       post '/v1/notice_of_disagreements',
-           params: VetsJsonSchema::EXAMPLES.fetch('NOD-CREATE-REQUEST-BODY_V1').to_json,
+           params: test_request_body.to_json,
            headers:
     end
 
+    let(:test_request_body) do
```

Comment:
Some notes for @data-doge after I did some investigation on why using the VetsJsonSchema example worked as a request body for these tests even though the structure is actually more aligned with the LH _response_ body:

- [Fixture](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/fixtures/notice_of_disagreements/valid_NOD_create_request.json) 🤝 [Request body in LH docs](https://dev-developer.va.gov/explore/api/decision-reviews/docs?version=current)
- [Example JSON](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/NOD-create-request-body_v1/example.json) 🙅 [Request body in LH docs](https://dev-developer.va.gov/explore/api/decision-reviews/docs?version=current)
- Seems like VCR doesn't care what the request body looks like — it just stubs out the response for whatever call is made to the LH endpoint when the test runs. 
  - I tried passing in empty params and it got past that API call, BUT fails at the new tests checking that the related evidence upload records are created because there were no `nodUploads` defined in the empty params 👍 if I add _only_ the upload params, the tests pass. So, more confirmation that the params didn't matter in the original build of the endpoint/tests.
  - The original code in the `create` endpoint directly passed the request body into the call to LH and didn't do anything else with it, so no real opportunities for the code to get caught on a structural difference.
- I compared the structure in [`schema.json`](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/NOD-create-request-body_v1/schema.json#L851) with [`example.json`](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/NOD-create-request-body_v1/example.json#L2) and the schema seems to match our fixture's structure instead of the example JSON, so even if there's any validation involving the schema (though I'm not sure there is), we should still be in the clear, as we are currently using that structure anyway.

Reply:
Here's the ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/70357